cirq/ops/raw_types.py

```diff
@@ -153,6 +153,48 @@ def _json_dict_(self):
         return protocols.obj_to_dict_helper(self, ['qubit', 'dimension'])
 
 
+class Tag:
```

Comment:
What value does this class add over simply having the tags be an `Any`?

Reply:
I guess you could technically do it that way, but that would be untyped and very error-prone.  It would not help the user figure out how to use the feature and you could do all sorts of non-sensical stuff, like add a circuit as a tag.